resources/aws/errors.go

```diff
@@ -13,6 +13,7 @@ var (
 
 	gatewayFindError       = errgo.New("Couldn't find gateway")
 	routeTableFindError    = errgo.New("Couldn't find route table")
+	routeFindError         = errgo.New("Couldn't find route")
```

Comment:
Also, I created [an issue](https://github.com/giantswarm/aws-operator/issues/123) to address this for other errors.